cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/RatioTapChangerConversion.java

```diff
@@ -116,7 +116,7 @@ private void addStepsFromTable(RatioTapChangerAdder rtca) {
         table.sort(byStep);
```

Comment:
Maybe we should return a `missing` warning here if `table` is empty like in `PhaseTapChangerConversion` ?

Reply:
Added